src/app/issuer/components/badgeclass-edit-form/badgeclass-edit-form.component.ts

```diff
@@ -401,12 +403,12 @@ export class BadgeClassEditFormComponent extends BaseAuthenticatedRoutableCompon
 		// update badge frame when a category is selected, unless no-hexagon-frame checkbox is checked
 		this.badgeClassForm.rawControl.controls['badge_category'].statusChanges.subscribe((res) => {
 			this.handleBadgeCategoryChange();
-			if(that.imageField.control.value){
+			if(that.imageField && that.imageField.control.value){
 				setTimeout(() => {
 					that.adjustUploadImage(this.badgeClassForm.value);
 				}, 10)
 			}
-			else if(that.customImageField.control.value){
+			else if(that.imageField && that.customImageField.control.value){
```

Comment:
Same as above: optional chaining makes this more readable.

Reply:
updated